src/actions/l10n.js

```diff
@@ -52,8 +58,9 @@ export function setupLocalization(
     });
 
     const fetchedMessages = await fetchMessages(languages[0]);
-    const bundles = lazilyParsedBundles([fetchedMessages]);
+    const bundles = lazilyParsedBundles([fetchedMessages], pseudoStrategy);
     const localization = new ReactLocalization(bundles);
-    dispatch(receiveL10n(localization));
+    const direction = getLocaleDirection(navigator.language, pseudoStrategy);
```

Comment:
Implementation note: I decided to store the direction in the redux state, instead of storing the pseudostrategy. I think both could be possible, but that's the 2 possibilities are equal.

Reply:
They're _not_ the same values, because `primaryLocale` comes from the language negotiation. If my prefered language is `ar` (an RTL language), but we don't have this translation, we'd fallback to english, and in that case we definitely want LTR and not RTL.

So using `primaryLocale` is actually the right thing to do here, thanks for pointing it out! This is because I used `navigator.language` before and forgot to change it here.